Gemfile

```diff
@@ -22,7 +22,7 @@ gem 'hiredis'
 gem 'http_accept_language'
 gem 'identity-doc-auth', github: '18F/identity-doc-auth', tag: 'v0.3.0'
 gem 'identity-hostdata', github: '18F/identity-hostdata', tag: 'v0.4.1'
-require File.join(__dir__, 'app', 'services', 'lambda_jobs', 'git_ref.rb')
+require File.join(__dir__, 'lib', 'lambda_jobs', 'git_ref.rb')
```

Comment:
We need to update the Dockerfile that points to this too: https://github.com/18F/identity-idp/blob/master/development.Dockerfile#L12